HomeTask2/rentalCarCost.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * This function calculates the totalCost of rent of a car
+ * @param {number} numberOfDays
+ * @returns {number} totalCost of the rent for the numberOfDays
+ * 
+ * @example rentalCarCost(7)
+ */
+function rentalCarCost(numberOfDays) {
+    const COST_PER_DAY = 40;
+    const DISCOUNT_FOR_7DAYS = 50;
+    const DISCOUNT_FOR_3DAYS = 20;
+    let totalCost = numberOfDays * COST_PER_DAY;
+    if (numberOfDays >= 7) {
+        return totalCost - DISCOUNT_FOR_7DAYS;
+    }
+    if (numberOfDays >= 3) {
+        return totalCost - DISCOUNT_FOR_3DAYS;
+    }
+    return totalCost;
```

Comment:
Do not use multiple return statements in a single function. Save it under one variable and return it at the end of the function.

Reply:
I made the changes as requested.
1. removed eval from basicOp
2.  removed multiple returns from rentalCarCost